src/config/abi/sousChef.json

```diff
@@ -165,5 +165,58 @@
     "outputs": [],
     "stateMutability": "nonpayable",
     "type": "function"
+  },
```

Comment:
para san to?

Reply:
kuys @medardm para sa mga bagong calls sa contract